omnij-rpc/src/integ/groovy/foundation/omni/test/rpc/sto/SendToOwnersTestPlanSpec.groovy

```diff
@@ -410,10 +421,14 @@ class SendToOwnersTestPlanSpec extends BaseRegTestSpec {
         }
     }
 
+    void assertBalance(Address address, CurrencyID currency, OmniValue expectedAvailable, OmniValue expectedReserved) {
+        assertBalance(address,currency,expectedAvailable.bigDecimalValue(),expectedReserved.bigDecimalValue())
+    }
+
     /**
      * Short cut to confirm a balance.
      */
-    void assertBalance(Address address, CurrencyID currency, def expectedAvailable, def expectedReserved) {
+    void assertBalance(Address address, CurrencyID currency, BigDecimal expectedAvailable, BigDecimal expectedReserved) {
```

Comment:
I think this was the change that found the Groovy bug. (Going from `def` to `BigDecimal`)